### PR TITLE
[Ubuntu][Windows] Remove announcements from generation process

### PR DESCRIPTION
**Pull request by vsafonkin**
# Description
Announcements table is removed from generation process to avoid appearing it in PR description. Announcements will be added to the readme files later, when the commit is creating.

#### Related issue:
internal issue [#1462](https://github.com/actions/virtual-environments-internal/issues/1462)

## Check list
- [x] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

**Review by maxim-lobanov — APPROVED** [review]

**Review by dibir-magomedsaygitov — APPROVED** [review]

**Review by AlenaSviridenko — APPROVED** [review]

**Review by al-cheb — APPROVED** [review]

**Comment by asklar** [comment]
I'm curious, with folks running into unforeseen breaking changes, why are announcements being removed instead of making sure that the issues mentioned in this file are easy to find?

**Comment by AlenaSviridenko** [comment]
Hi @asklar, anncouncements still will be included to Readme files and release descriptions, they just won't be published on PR description to avoid noise like this one
image

probable we need to update this PR title

File: images/linux/scripts/SoftwareReport/SoftwareReport.Generator.ps1 [hunk 1]
```diff
@@ -21,13 +21,6 @@ Restore-UserOwner
 
 $markdown = ""
```
Comment (MaksimZhukov) [inline images/linux/scripts/SoftwareReport/SoftwareReport.Generator.ps1:22]: Does it make sense to get rid of this line in both `SoftwareReport.Generator.ps1` files?

**Pull request merged**